fetch/worker.go

```diff
@@ -170,10 +176,15 @@ func getTxResultFromBatch(blocks []*types.Block, client Client) ([][]*types.TxRe
 
 		height := results.Height
 		deliverTxs := results.Results.DeliverTxs
+		blockIndex, exist := indexOfBlockHeight[height]
+
+		if !exist {
+			continue
+		}
```

Comment:
Is this situation even possible?

Reply:
This is not possible. 

As long as the RPC doesn't give us an incorrect height, we'll never run this conditional.

Reply:
Remove condition checks in this commit. https://github.com/gnolang/tx-indexer/pull/52/commits/c51e3d9eca5d1219e6d416d101ad5b6b4796f084